DockerMakefiles/CLI.yml

```diff
@@ -1,33 +1,26 @@
 opsin:
   requires:
-    - deploybase
+    - python_deploy_base
   description: IUPAC nomenclature parser, see doi:10.1021/ci100384d
   build: |
-    RUN apt-get update \
-      && apt-get install -y default-jre --no-install-recommends \
-      && cleanapt
-    RUN mkdir -p /opt/opsin \
-      && cd /opt/opsin \
-      && wget -nv https://bitbucket.org/dan2097/opsin/downloads/opsin-2.2.0-jar-with-dependencies.jar
-    RUN cmd='java -jar /opt/opsin/opsin-2.2.0-jar-with-dependencies.jar' \
-      && cmdf='#!/bin/bash\n'$cmd' $@\n' \
-      && printf "$cmdf" > /usr/local/bin/opsin \
-      && chmod +x /usr/local/bin/opsin
+    RUN conda install -qy -c bioconda opsin=2.1.0
+
+
+ambertools:
+  FROM: continuumio/miniconda3:4.3.11
+  build: |
+    RUN python -c "print('name = molecular-design-toolkit-builder\ninstitution = Autodesk\ncity = San Francisco\nstate or province = CA\ncountry = USA')" > /root/.amberrc
+
+    RUN conda install -qy -c ambermd ambertools=16.21.1
```

Comment:
Thanks @hainm ! Actually, MDT, really needs to update all of its version-pinned dependencies. Will track this in #144